docs/user-guide/command-line-interface.md

```diff
@@ -47,7 +47,7 @@ Specifying rules and plugins:
 Fixing problems:
   --fix                          Automatically fix problems
   --fix-dry-run                  Automatically fix problems without saving the changes to the file system
-  --fix-type Array               Specify the types of fixes to apply (problem, suggestion, layout)
+  --fix-type Array               Specify the types of fixes to apply (directive, problem, suggestion, layout)
```

Comment:
We should also update [`--fix-type` section](https://github.com/eslint/eslint/blob/master/docs/user-guide/command-line-interface.md#--fix-type) in the same document.